BurnerB/PropertyProLite-CH2
# [#167287887] A user can post a property

#### What does this PR do?
Allows an ADMIN to post a property
#### Description of Task to be completed?
* This route should only be accessible for an admin with an authenticated token.
* The property details should be saved in a database
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/PropertyProLite-CH2.git and cd into Propertypro-ch2
* checkout to branch ft-post-property-db-167287887
* Run command `npm start` on your terminal
* Using [postman](https://www.getpostman.com/)
* Navigate to `POST:api/v1/property` to send a valid payload;
* You should recieve a response:
```
{
	"status": 201,
    "data": {
        "owner": 2,
        "ownerEmail": "hezron3@gmail.com",
        "status": "Available",
        "type": "1 bedroom",
        "state": "Rwanda",
        "city": "Kigali",
        "price": "5000000",
        "address": "Kigali12345",
        "image_url": "http://res.cloudinary.com/burnerb/image/upload/v1563450179/nuez7l8kzjizntefbdpz.png"
    }
}

```
#### Any background context you want to provide?
This route is now configured to work with a database
#### What are the relevant pivotal tracker stories?
[#167287830](https://www.pivotaltracker.com/story/show/167287830)

#### Screenshots (if appropriate)
![postPROPERTYDBB](https://user-images.githubusercontent.com/45785786/61454924-e2e81980-a96a-11e9-8187-1dc974f16d0f.png)
